docs/controls/AdaptiveGridView.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: AdaptiveGridView XAML Control
+title: AdaptiveGridView
 author: nmetulev
 description: The AdaptiveGridView Control presents items in a evenly-spaced set of columns to fill the total available display space.

```

Comment:
```suggestion
description: The AdaptiveGridView Control presents items in an evenly-spaced set of columns to fill the total available display space.

```

Reply:
The GitHub web interface shows this suggestion as not applied but accessing the file directly shows the change has been applied.